src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -23,7 +23,7 @@
         "symfony/http-foundation": "~2.7|~3.0",
         "symfony/http-kernel": "~2.7|~3.0",
         "symfony/filesystem": "~2.7|~3.0",
-        "symfony/routing": "~2.7|~3.0",
+        "symfony/routing": "~3.0",
```

Comment:
why is it required for tests of the FrameworkBundle ? Which test is actually breaking when running it on Routing 2.7 ?